.github/workflows/macos.yml

```diff
@@ -1,18 +1,27 @@
-name: macOS
-
```

Comment:
any reasons to remove the name?

Reply:
Oh yeah i was playing with the badge and trying to make it work, undone now